### DISCUSSION TITLE
notes/chris-coyier/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#14] [Announcements]
# notes/chris-coyier/

Notes about the cascadiajs 2022 conference

https://cascadiajs.dendron.so/notes/chris-coyier/

**Comment by kevinslin** [comment]
great talk. something we've started using for image hosting is https://imagekit.io which is like cloudinary but with better pricing and focused only on images

**Comment by vim-usds** [comment]
Slides are here - https://drive.google.com/file/d/1b-eLtUQEUlb3xHJJ1TQVUol-2yjiUIFi/view?usp=sharing
